clang/test/Sema/constant_builtins_vector.cpp

```diff
@@ -723,3 +723,9 @@ not within the bounds of the input vectors; index of -1 found at position 0 is n
 permitted in a constexpr context}}
         vector4charConst1,
         vector4charConst2, -1, -1, -1, -1);
+
+static_assert(__builtin_reduce_add((vector4char){}) == 0);
```

Comment:
How is this handled? I'd have expected `Source.getVectorElt(0).getInt()` to have failed?